Applications/Spire/Source/Ui/CalendarDatePicker.cpp

```diff
@@ -198,6 +299,33 @@ CalendarDatePicker::CalendarDatePicker(
     new MonthSpinner(std::make_shared<RequiredDateModel>(m_model), this);
   layout->addWidget(month_spinner);
   layout->addWidget(make_day_header(this));
+  auto calendar_view = new ListView(
+    std::make_shared<CalendarListModel>(month_spinner->get_model()),
+    [=] (const std::shared_ptr<ListModel>& model, int index) {
+      return new CalendarDayLabel(
+        std::make_shared<ListValueModel>(model, index),
+        month_spinner->get_model());
+    }, this);
+  calendar_view->setFixedSize(scale(168, 144));
+  setFocusProxy(calendar_view);
+  month_spinner->setFocusProxy(calendar_view);
+  auto calendar_style = StyleSheet();
+  calendar_style.get(Any()).
+    set(Qt::Horizontal).
+    set(EdgeNavigation(EdgeNavigation::CONTAIN)).
+    set(Overflow(Overflow::WRAP));
+  calendar_style.get(Any() >> is_a<ListItem>()).
+    set(border_size(0)).
+    set(padding(0));
+  calendar_style.get(Any() >> (is_a<ListItem>() && Hover())).
+    set(BackgroundColor(QColor::fromRgb(0xFFFFFF)));
```

Comment:
Replace QColor::fromRgb(0xFFFFFF) with QColor(0xFFFFFF). I think the use of fromRgb will be discouraged.

Reply:
I used Qt::black for black because there's ambiguity between the QColor(const char *name) and QColor(QRgb) constructors when passing 0x000000.